src/tests/access/test_dual_ownable.cairo

```diff
@@ -86,22 +86,23 @@ fn test_dual_owner_exists_and_panics() {
 #[test]
 fn test_dual_transfer_ownership() {
     let (dispatcher, target) = setup_snake();
-    set_contract_address(OWNER());
+    start_cheat_caller_address(dispatcher.contract_address, OWNER());
     dispatcher.transfer_ownership(NEW_OWNER());
 
     let current_owner = target.owner();
     assert_eq!(current_owner, NEW_OWNER());
 }
 
 #[test]
+#[ignore] // TODO: Enable test once it's supported
```

Comment:
These comments should explain why is the test ignored.

Reply:
Updated